src/app/api/reference-data/model/emptyObject.ts

```diff
@@ -10,5 +10,4 @@
  * Do not edit the class manually.
  */
 
-export interface EmptyObject { 
-}
```

Comment:
Is this change needed?

Reply:
This was automatically changed by Biome. I'll roll this back and will add the API folder to the exclusion list of the linter to stop this happening again.